server/services/RunService.js

```diff
@@ -44,7 +43,269 @@ async function configureRuns({
     apg_example_ids,
     at_browser_pairs
 }) {
+    // if (!t) {
+    //     t = await db.sequelize.transaction();
+    // }
     try {
+        // TODO: add active to TestVersions model
+        // TODO: Get the active test version. if the test version has changed:
+        //         mark all the runs from the old test version as inactive
+        //         mark all the old apg_examples as inactive
+
+        // Activate/deactive APGExample rows
+
+        const apgExamples = await db.ApgExample.findAll({
+            where: {
+                test_version_id
+            }
+        });
```

Comment:
Look at this findAll, @rmeritz  -- if the config causes a change to the test version, we DO need to deactivate all the APGExamples associated with that old test version, because we aren't going to deactivate them in this logic. Same with the runs.

Reply:
we could change this to where active=true or where test_version_id